apps/emqx/test/emqx_session_SUITE.erl

```diff
@@ -126,34 +126,34 @@ t_unsubscribe(_) ->
 t_publish_qos0(_) ->
     ok = meck:expect(emqx_broker, publish, fun(_) -> [] end),
     Msg = emqx_message:make(clientid, ?QOS_0, <<"t">>, <<"payload">>),
-    {ok, [], Session} = emqx_session:publish(1, Msg, Session = session()),
-    {ok, [], Session} = emqx_session:publish(undefined, Msg, Session).
+    {ok, [], Session} = emqx_session:publish(clientinfo(),  1, Msg, Session = session()),
```

Comment:
nit: Extra spaces on several of these changed lines in this file

Reply:
I'll create another pull request to remove extra spaces, in the whole project.